WordPress/gutenberg
# [Mobile] EmptyList now uses `useEditorWrapperStyles` hook and removes `ReadableContentView`

Closes https://github.com/wordpress-mobile/gutenberg-mobile/issues/5618

**Related PRs:**
- https://github.com/wordpress-mobile/gutenberg-mobile/pull/5764

## What?
Follow up of https://github.com/WordPress/gutenberg/pull/50447 it updates the last component pending to use the `useEditorWrapperStyles` hook.

## Why?
The `EmptyList` component in the BlockList was still using the `ReadableContentView` component.

## How?
By updating the component to use the `useEditorWrapperStyles` hook. It also removes the `ReadableContentView` component from the codebase.

## Testing Instructions

- Open the editor
- Add a Group block
- Check the appender shows correctly
- Change the orientation to landscape
- Change the alignment of the block to full-width
- **Expect** the appender to adjust to the new block's width size.

### Testing Instructions for Keyboard
N/A

## Screenshots or screencast 
Portrait | Lanscape (Full-width alignment)
-|-
<kbd><img src="https://github.com/WordPress/gutenberg/assets/4885740/807c1dd1-fad7-42c0-8d85-bff8128455af" width=200 /></kbd> | <kbd><img src="https://github.com/WordPress/gutenberg/assets/4885740/de679331-de70-4d14-8d23-d7e06e59fa04" width=400 /></kbd>
